plugins/plugin-webpack/test/plugin.test.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+const fs = require('fs');
+
+const plugin = require('../plugin');
+
+test('minimal - no options', async () => {
```

Comment:
Not a request for change, but Jest makes it helpful to group tests with `describe()` for the group, and `it()` for the individual test. But here, this is fine—no grouping necessary.

Reply:
Also I like the test description here: `minimal - no options`. I’m +1 in general for very short, to-the-point test names which are much easier to read than a long, long list of complete sentences with complex structure.